src/parsers/csvParser.ts

```diff
@@ -0,0 +1,14 @@
+import { parse } from 'csv-parse/sync';
+import { promises as fs } from 'fs';
+
+export class CsvParser {
+
+    public static async parseCSV(filePath: string): Promise<any> {
+        const content = await fs.readFile(filePath);
+
+        return parse(content, {
+            columns: true,
+            skip_empty_lines: true
+        });
```

Comment:
I think it'd be better to make parse settings overridable if the user has a CSV file that does not work with these.

Reply:
Good point, will add a TODO since not a priority to start.